bin/spark-connect

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Since these scripts are aimed at development, maybe put them in connect/connect/bin?